### ISSUE TITLE
Sometimes dictionary words are temporarily flagged

### ISSUE THREAD

**Issue by cmichi** [drahnr/cargo-spellcheck#163]
**Describe the bug**

We run `cargo-spellcheck` 0.7.1 in our GitLab CI environment with the command
```
cargo spellcheck check --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1
```
In some runs, typos for words which are in the dictionary are detected. If the job is re-run it will succeed at one point. I've added `-vvvv` as debug output in the following runs. These are all for the same commit, just re-run.

* Success: https://gitlab.parity.io/parity/ink/-/jobs/856795
* Failure: https://gitlab.parity.io/parity/ink/-/jobs/856770
* Failure with a different file: https://gitlab.parity.io/parity/ink/-/jobs/856809

**To Reproduce**

I haven't managed to reproduce it locally, but I can easily reproduce it in our CI.

**Expected behavior**

Each run of `cargo-spellcheck` should be idempotent.

**Please complete the following information:**
 - System: Ubuntu 20.04
 - Obtained: `cargo install`
 - Version: 0.7.1

**Comment by drahnr** [comment]
Closed the wrong one...

**Comment by drahnr** [comment]
I can't seem to reproduce this, with the `ink` repo, I ran it 100 times and the output was identical each and every time with `v0.8.2`

**Comment by cmichi** [comment]
@drahnr Thanks for looking into it. I also didn't see this issue appear in recent times, so I'll close this issue and re-open if it appears again. Seems like it was fixed through something ✨ 🔮.
